Doc/whatsnew/3.8.rst

```diff
@@ -1586,9 +1569,9 @@ Changes in the Python API
   specialized methods like :meth:`~tkinter.ttk.Treeview.selection_set` for
   changing the selection.  (Contributed by Serhiy Storchaka in :issue:`31508`.)
 
-* The :meth:`writexml`, :meth:`toxml` and :meth:`toprettyxml` methods of the
-  :mod:`xml.dom.minidom` module, and :mod:`xml.etree` now preserve the attribute
-  order specified by the user.
+* The :meth:`writexml`, :meth:`toxml` and :meth:`toprettyxml` methods of
+  :mod:`xml.dom.minidom`, and the :func:`write` method of the :mod:`xml.etree` module,
```

Comment:
Inconsistent change.  'module' is removed from xml.dom.mindom and added to xml.etree.

Reply:
It reads best to me to take out the word "module"

(Also, looking at it again, I noticed that :func:`write` should really be :meth:.  I'll fix that too.)